examples/model_selection/plot_roc.py

```diff
@@ -1,10 +1,10 @@
 """
-=======================================
-Receiver Operating Characteristic (ROC)
-=======================================
+==================================================
+Multiclass Receiver Operating Characteristic (ROC)
+==================================================
 
-Example of Receiver Operating Characteristic (ROC) metric to evaluate
-classifier output quality.
+Example of Receiver Operating Characteristic (ROC) metric to evaluate multiclass
+classifiers quality.
```

Comment:
Perhaps just a matter of taste but what about making a true sentence here? Something like:
"This example describes the use of the Receiver Operating Characteristic (ROC) metric to evaluate multiclass
classifiers quality."